app/controllers/users/users_controller.rb

```diff
@@ -0,0 +1,30 @@
+class Users::UsersController < ApplicationController
```

Comment:
this file should be app/controllers/api/users_controller.rb
and rendering json views????